x-pack/plugins/cloud_security_posture/public/pages/findings/latest_findings/latest_findings_container.tsx

```diff
@@ -64,31 +72,49 @@ export const LatestFindingsContainer = ({ dataView }: FindingsBaseProps) => {
         setQuery={(query) => {
           setUrlQuery({ ...query, pageIndex: 0 });
         }}
-        query={urlQuery.query}
-        filters={urlQuery.filters}
-        loading={findingsCount.isFetching}
+        loading={findingsGroupByNone.isFetching}
       />
       <PageWrapper>
         <LatestFindingsPageTitle />
         <FindingsGroupBySelector type="default" />
-        {findingsDistribution && <FindingsDistributionBar {...findingsDistribution} />}
-        <EuiSpacer />
-        <FindingsTable
-          data={findingsGroupByNone.data}
-          error={findingsGroupByNone.error}
-          loading={findingsGroupByNone.isFetching}
-          pagination={getPaginationTableParams({
-            pageSize: urlQuery.pageSize,
-            pageIndex: urlQuery.pageIndex,
-            totalItemCount: findingsGroupByNone.data?.total || 0,
-          })}
-          sorting={{
-            sort: { field: urlQuery.sort.field, direction: urlQuery.sort.direction },
-          }}
-          setTableOptions={({ page, sort }) =>
-            setUrlQuery({ pageIndex: page.index, pageSize: page.size, sort })
-          }
-        />
+        {error && <ErrorCallout error={error} />}
```

Comment:
I think you should also hide the group-by selector in this case, it is awkward on the screen and when selecting different options the query still persists, which leads to another error view. So the user has to fix the query before selecting different group-by options, meaning we should hide this control until the query is fixed.